{"uuid":"94aeae8d-8b8c-4545-abb5-63584e91da85","historyId":"2177751271a8985b43d563327f3e069e","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"dsAlgoRunner.Runner_Test"},{"name":"testClass","value":"dsAlgoRunner.Runner_Test"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"dsAlgoRunner.Runner_Test"},{"name":"host","value":"WINDOWS-I321LLC"},{"name":"thread","value":"25272@WINDOWS-I321LLC.TestNG-PoolService-2(26)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'WINDOWS-I321LLC', ip: '10.0.0.57'\nBuild info: version: '4.16.1', revision: '9b4c83354e'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [], binary: C:\\Program Files\\Google\\Chr..., extensions: []}, pageLoadStrategy: normal}]}]","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'WINDOWS-I321LLC', ip: '10.0.0.57'\nBuild info: version: '4.16.1', revision: '9b4c83354e'\nSystem info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.2'\nDriver info: org.openqa.selenium.chrome.ChromeDriver\nCommand: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [], binary: C:\\Program Files\\Google\\Chr..., extensions: []}, pageLoadStrategy: normal}]}]\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:237)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:164)\r\n\tat org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:114)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:89)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:84)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:73)\r\n\tat dsutilities.DriverFactory.inint(DriverFactory.java:25)\r\n\tat dsalgoHooks.DsHooks.launchbrowser(DsHooks.java:27)\r\nCaused by: java.io.UncheckedIOException: java.io.IOException: Cannot run program \"C:\\Users\\gayathri\\.cache\\selenium\\chromedriver\\win64\\124.0.6367.155\\chromedriver.exe\": CreateProcess error=32, The process cannot access the file because it is being used by another process\r\n\tat org.openqa.selenium.os.ExternalProcess$Builder.start(ExternalProcess.java:193)\r\n\tat org.openqa.selenium.remote.service.DriverService.start(DriverService.java:208)\r\n\tat org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:119)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:523)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:237)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:164)\r\n\tat org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:114)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:89)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:84)\r\n\tat org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:73)\r\n\tat dsutilities.DriverFactory.inint(DriverFactory.java:25)\r\n\tat dsalgoHooks.DsHooks.launchbrowser(DsHooks.java:27)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat io.cucumber.java.Invoker.doInvoke(Invoker.java:66)\r\n\tat io.cucumber.java.Invoker.invoke(Invoker.java:24)\r\n\tat io.cucumber.java.AbstractGlueDefinition.invokeMethod(AbstractGlueDefinition.java:47)\r\n\tat io.cucumber.java.JavaHookDefinition.execute(JavaHookDefinition.java:64)\r\n\tat io.cucumber.core.runner.CoreHookDefinition.execute(CoreHookDefinition.java:46)\r\n\tat io.cucumber.core.runner.HookDefinitionMatch.runStep(HookDefinitionMatch.java:21)\r\n\tat io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)\r\n\tat io.cucumber.core.runner.TestStep.executeStep(TestStep.java:81)\r\n\tat io.cucumber.core.runner.TestStep.run(TestStep.java:53)\r\n\tat io.cucumber.core.runner.TestCase.run(TestCase.java:78)\r\n\tat io.cucumber.core.runner.Runner.runPickle(Runner.java:75)\r\n\tat io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:132)\r\n\tat io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:136)\r\n\tat io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)\r\n\tat io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:136)\r\n\tat io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)\r\n\tat io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:35)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)\r\n\tat org.testng.internal.invokers.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:80)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\nCaused by: java.io.IOException: Cannot run program \"C:\\Users\\gayathri\\.cache\\selenium\\chromedriver\\win64\\124.0.6367.155\\chromedriver.exe\": CreateProcess error=32, The process cannot access the file because it is being used by another process\r\n\tat java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)\r\n\tat java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)\r\n\tat org.openqa.selenium.os.ExternalProcess$Builder.start(ExternalProcess.java:191)\r\n\t... 40 more\r\nCaused by: java.io.IOException: CreateProcess error=32, The process cannot access the file because it is being used by another process\r\n\tat java.base/java.lang.ProcessImpl.create(Native Method)\r\n\tat java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:500)\r\n\tat java.base/java.lang.ProcessImpl.start(ProcessImpl.java:159)\r\n\tat java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)\r\n\t... 42 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"\"F3 The signedin user selects the  stack datastructure module and runs the python code\""},{"name":"arg0","value":"\"The user clicks the Stack topic links and runs the  invalid python code\""}],"start":1715108816676,"stop":1715108820657}